client/src/components/NavBar/index.js

```diff
@@ -1,12 +1,25 @@
 import React from "react";
 import "./NavBar.css";
 import logo from "./logo.png";
+import axios from 'axios';
 
-const NavBar = () => (
+
+function handleSubmit(props){
+
+  const { history } = props;
+  axios
+    .get("/logout")
+    .then(() => {
+      history.push("/");
+    })
+    .catch(error => console.log(error));
```

Comment:
Render ```error``` by redirect the user to Error page

Reply:
done :+1: